domain/servicetemplate/templatestore.go

```diff
@@ -76,8 +76,23 @@ func (s *storeImpl) Delete(ctx datastore.Context, id string) error {
 // GetServiceTemplates returns all ServiceTemplates
 func (s *storeImpl) GetServiceTemplates(ctx datastore.Context) ([]*ServiceTemplate, error) {
 	q := datastore.NewQuery(ctx)
-	query := search.Query().Search("_exists_:ID")
-	search := search.Search("controlplane").Type(kind).Query(query)
+
+	query := map[string]interface{}{
+		"query": map[string]interface{}{
+			"bool": map[string]interface{}{
+				"must": []map[string]interface{}{
+					{"exists": map[string]string{"field": "ID"}},
+					{"term": map[string]string{"type": kind}},
```

Comment:
Can't these exists queries now just be queries for kind?

Reply:
I'm not sure whether the ID field is obligatory for the servicetemplate type. If ID is obligatory then we can remove the part with exist from the query

Reply:
ID should be required on most objects. If there is no performance difference I wouldn't bother changing